### ISSUE TITLE
Gpay / Digital wallet, Payments API response issue

### ISSUE THREAD

**Issue by savaskalkan** [Fiserv/Support#126]
# Reporting new issue for Gpay

**Describe the bug**
When we use Gpay in our React Native mobile app, we are getting the following error after Fiserv payments API.
it says both parameters can not be empty but in the sample payload, these parameters already don't exist.

code: "INVALID_INPUT
field: "authenticationRequest, authenticationResult"
message: "authenticationRequest and authenticationResult fields cannot both be empty."

**To Reproduce**
1. Use Gpay
2. Send digital wallet data to "/gateway/v2/payments"
3. And you will have 400 responses and errors like following
code: "INVALID_INPUT
field: "authenticationRequest, authenticationResult"
message: "authenticationRequest and authenticationResult fields cannot both be empty."

**Expected behavior**

**Screenshots**

<img width="845" alt="image" src="https://user-images.githubusercontent.com/12950142/163384232-6b5dcf40-a02b-4055-ae7c-a9aaaf96110d.png">

**Smartphone (please complete the following information):**
 - Device: Samsung A21s
 - OS: Android 11

**Additional context**
I am adding this response data as text to help
data:
apiTraceId: "YlgEhQX6UcEUCmNH4qjnLgAAArQ"
clientRequestId: "d0005777-4383-4c3c-afe9-190ebbc77cf4"
error:
code: "INVALID_INPUT"
details: Array(1)
0:
field: "authenticationRequest, authenticationResult"
message: "authenticationRequest and authenticationResult fields cannot both be empty."
[[Prototype]]: Object
length: 1
[[Prototype]]: Array(0)
message: "Invalid request input. Please see details below."
[[Prototype]]: Object
responseType: "BadRequest"
[[Prototype]]: Object
duration: 2733

    At the end of your Bug template, please mention team you would like to subscribe for this Issue for future updates. You can mention teams in Github 
    
    Example:  @Fiserv/fiserv-developer)

-->

**Comment by acho-fiserv** [comment]
thanks for posting @savaskalkan .

which development portal documentation were you using?

**Comment by savaskalkan** [comment]
Thank you @acho-fiserv
Actually, I don't get what you mean. But I used these documents 
https://docs.fiserv.com/docs/payments/b3A6NDc3NjE4NA-generate-a-primary-transaction
https://docs.firstdata.com/org/gateway/node/163
https://docs.firstdata.com/org/gateway/docs/api#digital-wallets

**Comment by acho-fiserv** [comment]
thanks @savaskalkan , that's what i'm looking for.

i'm waiting to hear back from some contacts.  stay tuned.

**Comment by acho-fiserv** [comment]
submit your request to the support team please at https://fiserv.dev/ since what you need is related to the EMEA APIs

**Comment by savaskalkan** [comment]
@acho-fiserv I've already left a message last week. And I've just left one more.

**Comment by savaskalkan** [comment]
@acho-fiserv this is urgent for our app because it is met in the production. Yesterday I wrote the support team but can you help it to reach a solution quickly if possible.

**Comment by karen791** [comment]
@acho-fiserv Please can you help asap in getting this issue resolved please as this is now becoming urgent! as it is service affecting

**Comment by karen791** [comment]
@acho-fiserv - Any progress please?

**Comment by acho-fiserv** [comment]
hi @karen791 , this is what i have so far

[1:32 AM] Lockyer, Matt (London)
    Hi Alvin, our support team already responded and is waiting on the product team to respond

**Comment by karen791** [comment]
Thank you for getting back to me! i there anyone in the product team I can contact about this please?

**Comment by fsmherrm** [comment]
Hello @savaskalkan , could you please try again? I have made a configuration change on our side.

**Comment by acho-fiserv** [comment]
closing due to inactivity
